editor/src/components/canvas/controls/select-mode/remix-scene-label.tsx

```diff
@@ -87,12 +90,36 @@ export const RemixSceneLabelControl = React.memo<RemixSceneLabelControlProps>((p
   )
 })
 
+const RemixScenePathUpdater = React.memo<{ targetPathString: string }>((props) => {
+  const [navigationData] = useAtom(RemixNavigationAtom)
+
+  const sceneDataLabelRef = useRefEditorState((store) =>
+    getRemixSceneDataLabel(store.editor.jsxMetadata[props.targetPathString]),
+  )
+
+  const updateRemixScenePathInLiveblocks = useUpdateRemixSceneRouteInLiveblocks()
+  const currentPath = (navigationData[props.targetPathString] ?? null)?.location.pathname
+  React.useEffect(() => {
+    if (sceneDataLabelRef.current == null || currentPath == null) {
+      return
+    }
+
+    updateRemixScenePathInLiveblocks({
+      sceneDataLabel: sceneDataLabelRef.current,
+      location: currentPath,
+    })
+  }, [currentPath, sceneDataLabelRef, updateRemixScenePathInLiveblocks])
+
+  return <></>
```

Comment:
why not `return null`?

Reply:
Great idea, I completely glossed over that, added it in https://github.com/concrete-utopia/utopia/pull/4744/commits/a9e0b34476318fc3248abfc575bea8775c9af71d